### DISCUSSION TITLE
Any help needed on UI ?

### DISCUSSION THREAD

**Discussion by falzfalzfalz** [pockethost/pockethost#129] [Q&A]
Hi @benallfree,

we talked briefly on the PocketBase github, and you said that you might use some help here. I have some spare time this week and would gladly help this project to grow as I looked into the project and started using it to host my own pocketbase projects.

I have quite some experience with UI and Svelte and found the features section of the website to be quite minimal and no footer whatsoever, would you be ok with a PR for an overhaul of this section/main page ?

Best,

Shaparder

**Comment by benallfree** [comment]
Hi @shaparder, thanks for the offer!

Yes, there are many frontend issues you can play with. Here are a few that are already open:

* FAQ page #37 
* Customize DB backups page #95 
* Dart and REST samples: #33 

Additionally, there are other upcoming cosmetics in #27 which is currently a work in progress.

Beyond these, you can also pick up what you suggested and organize the features better (including rewording/editing them if you wish). Privacy policy, ToS... there are quite a lot of things to do.

Let me know what you decide or if you have any questions!

**Reply by falzfalzfalz** [reply]
Hi @benallfree, sorry for the delay. I started working on some UI stuff ! I decided to start with the FAQ issue, I suggest we talk on the initial issue #37 ! See you there :)
